python-checks/src/main/java/org/sonar/python/checks/MethodShouldBeStaticCheck.java

```diff
@@ -106,6 +122,10 @@ private static boolean isDocstringOrPass(AstNode statement) {
   private static boolean isDocstringAndPass(AstNode statement1, AstNode statement2) {
     return statement1.getToken().getType().equals(PythonTokenType.STRING) && statement2.getFirstDescendant(PythonGrammar.PASS_STMT) != null;
   }
+  
+  private static boolean isRaise(AstNode statementList) {
+    return statementList.getFirstChild().getFirstDescendant(PythonGrammar.RAISE_STMT) != null;
```

Comment:
@ivandalbosco minor: you can replace `... getFirstDescendant(PythonGrammar.RAISE_STMT) != null` with `hasDescendant`